frontend/src/components/toolbar/problem-metadata/problem-metadata.component.tsx

```diff
@@ -53,20 +57,43 @@ export function ProblemMetadataComponent() {
   //#endregion
 
   //#region handlers
+
+  // TODO: toast the error messages instead of console logging
   const handleSubmit = async (event) => {
     event.preventDefault();
-    if (schedule) {
-      console.log(schedule);
-      const response = await backend.getErrors(schedule);
-      dispatcher({
-        type: ScheduleErrorActionType.UPDATE,
-        payload: response,
-      } as ActionModel<ScheduleErrorModel[]>);
+
+    if (!schedule?.employee_info?.nurseCount || !schedule?.employee_info?.babysitterCount) {
```

Comment:
Żeby nie pisać za każdym razem `schedule.....`, można było by przypisać to do zmiennych i potem używać w kodzie. To moim zdaniem było by trochę czytelniejsze. 

Btw, dziś przeczytałem o bardzo przyjemnej funkcjonalności js która pozwala zrobić takie przepisywanie do pół w bardzo ładny sposób:

```
obj = { 'a': 1, 'b': 2}
let { a,...rest} = obj
console.log(a)
>> 1
```
czyli można by było zrobić coś w styłu 
```
let { nurseCount, sitterCount ,...rest } = schedule?.employee_info || {}

```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment